opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -0,0 +1,211 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opencensus.proto.stats.metrics;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "metricsproto";
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.metrics";
+option java_outer_classname = "MetricsProto";
+
+// A collection of MetricPoint's.
+message MetricPointSet {
+  // Each MetricPoint has one or more timeseries for a single metric.
+  repeated MetricPoint metric_points = 1;
+}
+
+// Defines a MetricPoint which has one or more timeseries for a single metric.
+message MetricPoint {
```

Comment:
Shouldn't this be named MetricTimeSeries? MetricPoint makes it sound like it represents only one point.

Reply:
Why not just "Metric"?
I agree that "point" is confusing.

Reply:
went with Metric/MetricSet. Let me know if it makes sense.